### DISCUSSION TITLE
Can't find some elements in the documentation

### DISCUSSION THREAD

**Discussion by Haislich** [tracel-ai/burn#1881] [Q&A] [answered]
In the documentation some elements are missing and thus require to go through the source code.
An example is  [NamedDims](https://github.com/tracel-ai/burn/blob/main/crates/burn-tensor/src/tensor/named/dims.rs#L14) that is in one of the examples but [cannot be found by query search in the docs](https://burn.dev/docs/burn/index.html?search=NamedDims).
If the omission of some elements from the documentation is intentional I apologize.
If I'm doing the search wrong, I apologize and can you please explain what I'm missing ?
Else I would like, if possible,to contribute.
This would be my first contribution and I don't know where and how should I start.
Can somebody guide me  ?

**Comment by laggui** [comment, marked as answer]
`NamedDims` are still [experimental](https://github.com/tracel-ai/burn/issues/567) so that's why they are excluded when generating the docs.

Right now only the elements included in the `doc` feature flag are used to generate the documentation. You can check the corresponding features in the crates' Cargo.toml (e.g., [`burn-core`](https://github.com/tracel-ai/burn/blob/main/crates/burn-core/Cargo.toml#L42))
